### ISSUE TITLE
Define composer packages to install them globally

### ISSUE THREAD

**Issue by chrfritsch** [hashicorp/otto#229]
It would be fine to define some composer packages in the Appfile to install them globally.
Would be very helpful to install tools like phing or phpunit in the dev box.

**Comment by chad-thompson** [comment]
As a matter of a bit of discussion:  would it be better for most users to install composer libraries globally, or would it be better to maintain those dependencies in a separate composer.json file?  The thought:  it seems that the goal is to do something like keep your Otto `Appfile` version controlled, etc. with the rest of your PHP application - so a directory might look like:

```
Appfile
composer.json
vendor/
myphp/
etc/
```
It might be simpler (and I am certainly not an expert on this) for various development / authoring tools to continue to use the PHP composer format (`composer.json`) in case you are using IDEs or other development environments.

Is there a particular advantage to installing composer libraries globally?

**Comment by Swader** [comment]
> Is there a particular advantage to installing composer libraries globally?

Yes. It's common practice for specific tools that get used machine-wide, on multiple projects. Tools like testing frameworks, Phar packagers, composer itself, code linters, deploy / build tools, and so on. The configuration for each of those tools is local to the individual projects, but the tools themselves get installed globally. So project-specific libs go into local composer.json files, as usual, while those pertaining to the entire environment are usually installed globally.

Maintaining a separate composer.json file for global libraries is a good idea - have Appfile consider it while building the environment. That way we keep the standard composer syntax, but get both local and global composer package installation support.
